rcongui/Dockerfile

```diff
@@ -2,26 +2,30 @@ FROM node:13-stretch
 
 WORKDIR /code
 
-RUN apt-get update -y && apt-get install -y nginx apache2-utils
+RUN apt-get update -y && apt-get install -y yarn nginx apache2-utils
 COPY nginx.conf /etc/nginx/sites-available/default
 
 RUN mkdir /pw
 
 COPY package.json .
-COPY package-lock.json .
+COPY yarn.lock .
 
-RUN npm install
+RUN yarn install
```

Comment:
Is there really a difference here? npm improved a lot recently and yarn is from evil facebook :P

Reply:
NPM (two 'npm i' cmd) Done in 40.897s and Done in 31.601s
YARN (two 'yarn install' cmd) Done in 57.59s. Done in 0.41s.

'yarn is from evil facebook'  Yes... 🤣  but evil is already in the place (React 👀 ?)

Anyway, if you want want to keep npm, I can just leave 'RUN rm xxx' cmds 👍

Reply:
Eheh., indeed. 
Ye let's stick with npm please. I'm used to the tool :)

Reply:
done